Entities/MovableObject.cpp

```diff
@@ -134,6 +137,8 @@ int MovableObject::Create()
 	m_TerrainMatHit = g_MaterialAir;
 	m_ParticleUniqueIDHit = 0;
 
+    m_UpdateFrames = 0;
```

Comment:
I don't think you need to clear outside of clear. Clear is called when the object is destroyed (and returns to the memory pool), and the object doesn't accumulate frames to its counter until it exists again and is updating.